plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/messages.properties

```diff
@@ -7,7 +7,7 @@ fix.cloud.sdk.location=Cannot create an App Engine Eclipse project \
  and set the location in the Google Cloud Platform preferences.
 fix.cloud.sdk.version=Cannot create an App Engine Eclipse project \
  because the Cloud SDK is too old. \
- Update it by running `gcloud components update` at the command line.
+ Update it by running ''gcloud components update'' at the command line.
```

Comment:
What does this do?

Reply:
Changing ` to ' in the UI text. This also has an effect of making the quoted text bold in the error page. This actually makes the message consistent with the other error message that already uses single quotes.

```
 fix.appengine.java.component=...
  Fix by running ''gcloud components install app-engine-java'' on the command-line.
...
 fix.cloud.sdk.version=...
  Update it by running `gcloud components update` at the command line.
```

Reply:
FYI, the quote text automatically becoming bold is by our custom logic in the `MissingComponentPage`.